### ISSUE TITLE
Feature Request: Configure file extensions

### ISSUE THREAD

**Issue by jasonpatrickellykrause** [hilleer/vscode-yaml-plus-json#32]
I would like a "yaml-plus-json" toggle for renaming files to .yaml instead of .yml.

**Comment by jasonpatrickellykrause** [comment]
Something like this:

```JSON
    "yaml-plus-json": {
        "convertOnRename": true,
        "yamlIndent": 2,
        "yamlSchema": "core",
        "yamlExtension": "yaml",
        "jsonExtension": "json"
    },
    ```

**Comment by hilleer** [comment]
Hi @jasonpatrickellykrause ,

Interesting. I think that makes a lot of sense :-) I'll keep you updated on implementation of this.

Meanwhile, feel free to contribute and open a PR.

**Comment by jasonpatrickellykrause** [comment]
I will take a look when I have time.  Thinking more about it, I would like to expand to jsonc, but that is a different feature. 

https://code.visualstudio.com/docs/languages/json#_json-with-comments

**Comment by hilleer** [comment]
> I will take a look when I have time. Thinking more about it, I would like to expand to jsonc, but that is a different feature.
> 
> https://code.visualstudio.com/docs/languages/json#_json-with-comments

That kind of relates to this issue: https://github.com/hilleer/vscode-yaml-plus-json/issues/18.

I did not find a good way to support that yet.

**Comment by hilleer** [comment]
Hi @jasonpatrickellykrause

I have published a new version `1.7.0` implementing configurable filename extension. See [config in readme](https://github.com/hilleer/vscode-yaml-plus-json#config) for details.

FYI I have defaulted yaml filename to `.yaml` rather than `.yml` as that seemed to be the official recommended one.

Please don't hesitate to give feedback or open new feature requests.

Happy conversion!
